ratbag_emu/util.py

```diff
@@ -0,0 +1,26 @@
+# SPDX-License-Identifier: MIT
+
+
+def mm2inch(mm):
+    return mm * 0.0393700787
+
+
+def ms2s(ms):
+    return ms / 1000.0
+
+
+class EventData(object):
+    def __init__(self, x=0, y=0):
+        self.x = x
+        self.y = y
```

Comment:
hmm, not sure the generic `EventData` should have references to `x` and `y`. But we can keep this one for now.